Kudu.Core/Deployment/DeploymentManager.cs

```diff
@@ -544,6 +544,31 @@ private DeployResult GetResult(string id, string activeDeploymentId, bool isDepl
                 string repositoryRoot = _environment.RepositoryPath;
                 var perDeploymentSettings = DeploymentSettingsManager.BuildPerDeploymentSettingsManager(repositoryRoot, _settings);
 
+                string delayMaxInStr = perDeploymentSettings.GetValue(SettingsKeys.MaxRandomDelayInSec, false);
+                if (delayMaxInStr != null) {
+                    int maxDelay;
+                    if(!Int32.TryParse(delayMaxInStr, out maxDelay) || maxDelay < 0)
```

Comment:
just personal suggestion no convention i think.  it will be easier to explain and follow if we only accept the exact format, either number or timespam format. 

otherwise user who use this feature might get confuse. e.g value was set to "120s", so was expecting more than 1 minutes delay for some occasion, however current code convert to 60 seconds max internally. if user do not read the log they won`t notice this changes. but if there is no delay, user will come back and revisit what they did wrong.